app/serializers/node_info/serializer.rb

```diff
@@ -38,7 +38,10 @@ def open_registrations
   end
 
   def metadata
-    {}
+    {
+      nodeName: Setting.site_title,
+      nodeDescription: Setting.site_short_description,
```

Comment:
I assumed short is more common than site_extended_description ... correct me if that's wrong :)